project/gh-action/infrastructure/kubernetes/ingress.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: networking.k8s.io/v1beta1 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)